fe1-web/src/features/evoting/screens/CreateElection.tsx

```diff
@@ -74,7 +74,9 @@ const isQuestionInvalid = (question: NewQuestion): boolean =>
  * @param questions The list of questions
  */
 const haveQuestionsSameTitle = (questions: NewQuestion[]): boolean => {
-  const questionTitles = questions.map((q: NewQuestion) => q.question);
+  const questionTitles = questions
+    .map((q: NewQuestion) => q.question.trim())
```

Comment:
This is great, in the final questions will be trimmed and thus we should check for equality where they are trimmed. But since we now make a transformation between the data displayed to the user and what will be used to create the election we should ensure we perform the checks on the same data that will then be used to create the election. Otherwise the two things might differ and we have code duplication.

One way to solve is is by separating the values displayed to the user and the ones used for processing later. For instance you can create an additional `electionName` and `questions` variables and rename the old variables to something like `electionNameInput` and `questionsInput`.

E.g.
```typescript
const [questionsInput, setQuestionsInput] = useState<NewQuestion[]>([EMPTY_QUESTION]);
...


const questions = useMemo(() => {
  return questionsInput.map((q: NewQuestion) => {...q, question: q.question.trim()});
}, [questionsInput]);
```

In this case `questionsInput` represents what is entered by the user and `questions` what we use for processing. All checks should be performed on `questions` and tell the user what they need to change in `questionsInput` to make the input valid. `useMemo` updates its value if one of it's dependencies (in the array) change.

This way we're sure that the data used for processing (`questions`, derived from `questionsInput` ) is valid.

Does this make sense?